Tests/ConnectLibraryTests/ConnectTests/ConnectErrorTests.swift

```diff
@@ -87,17 +98,23 @@ final class ConnectErrorTests: XCTestCase {
 
     // MARK: - Private
 
-    private func errorData(expectedDetails: [ProtobufMessage]) throws -> Data {
+    private func errorData(expectedDetails: [ProtobufMessage], pad: Bool = true) throws -> Data {
         // Example error from https://connectrpc.com/docs/protocol/#error-end-stream
         let dictionary: [String: Any] = [
             "code": "unavailable",
             "message": "overloaded: back off and retry",
             "details": try expectedDetails.map { detail in
-                [
+                var val = try detail.serializedData().base64EncodedString()
+                // If pad is false, remove all the padding added by the base64EncodedString function
+                if !pad {
+                    val = val.replacingOccurrences(of: "=", with: "")
+                }
+                let errDetail = [
```

Comment:
```suggestion
                return [
```
Can just return this directly